gormigrate.go

```diff
@@ -327,7 +327,7 @@ func (g *Gormigrate) rollbackMigration(m *Migration) error {
 	}
 
 	if err := m.Rollback(g.tx); err != nil {
-		return err
+		return fmt.Errorf("migration-step %s: %w",m.ID, err)
```

Comment:
I think it should probably be something like `gormigrate: Migration "%s" failed: %w`

Reply:
done ;)